handler/getTeamMembers.go

```diff
@@ -0,0 +1,32 @@
+package handler
+
+import (
+	"github.com/forjadev/gun-organization/schemas"
+	"github.com/gin-gonic/gin"
+)
+
+// @Summary Returns the list of team members
+// @Description Returns the list of members of specific team
+// @Tags Teams
+// @Accept json
+// @Produce json
+// @Success 200 {object} ListTeamMembersResponse
+// @Router /team/{teamname}/members [get]
+func ListTeamMembersHandler(ctx *gin.Context) {
+	teamName := ctx.Param("teamname")
+
+	if teamName == "" {
+		sendError(ctx, 404, "Team name is required")
```

Comment:
Maybe we can use the http.StatusNotFound instead of the hardcoded http code for easy documentation and reading, what you think?